Catappult/appcoins-wallet-android
# Feature/ESKL-250 wallet payment flow wallet version ban

**What does this PR do?**
   Until now, when a user is blocked from creating a ticket because of its wallet version, it gets a generic error. 
   The user now receives an error asking it to update the Apcoins Wallet to the latest version.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] skills/src/main/res/layout/wallet_version_not_supported_layout.xml
- [ ] skills/src/main/java/cm/aptoide/skills/SkillsViewModel.kt

**How should this be manually tested?**
   Try to create a ticket with an old wallet version, you should get a message asking to update. Once you click on "Update" you should get Android Market options. (some changes may be necessary on E-Skills side to test this).

**What are the relevant tickets?**

  Tickets related to this pull-request: [ESKL-250](https://aptoide.atlassian.net/browse/ESKL-250)

**Code Review Checklist**
